### PR TITLE
Portfolio: Added form validation

### PR DESCRIPTION
**Pull request by amare1990**
- Add span small tag at the end of the contact form to display the error message.
- Add css styles for the error message for both the mobile and desktop version. 
- Write javascript codes to match to valid email value inputs using regular expression.
- Modify the javascript code algorithm to display errors when invalid email value is only inserted.

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -459,4 +459,39 @@ function showPopupWindow() {
...
+
+
+
+// Contact Form Validation
+
+const emailInput = document.getElementById('email-input');
+console.log(emailInput.value);
+let emailPattern =  /^[a-z0-9-]+@[a-z0-9]+\.[a-z0-9.]+$/;
+
+//console.log("node value= "+emailInputValue.value);
```
Comment (emmiiorji) [inline js/scripts.js]: Kindly remove commented pieces of code in order to make the code neater. You can always go back in commit history to find them.

File: js/scripts.js [hunk 2]
```diff
@@ -459,4 +459,39 @@ function showPopupWindow() {
 
 showPopupWindow(); 
 
-} //end of for desktop version
\ No newline at end of file
+} //end of popup window for desktop version
+
+
+
+
```
Comment (emmiiorji) [inline js/scripts.js]: Kindly avoid leaving unnecessary spaces in your code to make it more readable and concise.

File: js/scripts.js [hunk 3]
```diff
@@ -459,4 +459,39 @@ function showPopupWindow() {
...
+// Contact Form Validation
+
+const emailInput = document.getElementById('email-input');
+console.log(emailInput.value);
```
Comment (emmiiorji) [inline js/scripts.js]: Please, remove console.log statements after debugging so that your code will be more concise.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @amare1990, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a little more


## Good Highlights ✔

- Adherence to GitHub flow
- Good commit messages
- Linter checks are passing


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @amare1990 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Removed Comments✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

*************************************
# Optional Suggestions 

- [ ] N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**